apps/ledger-live-mobile/src/newArch/features/Web3Hub/components/ManifestsList/useManifestsListViewModel.ts

```diff
@@ -0,0 +1,24 @@
+import { useInfiniteQuery } from "@tanstack/react-query";
+import {
+  fetchManifestsMock,
+  selectManifests,
+  getNextPageParam,
+} from "LLM/features/Web3Hub/utils/api/manifests";
+
+export default function useManifestListViewModel(selectedCategory: string) {
+  const manifestsQuery = useInfiniteQuery({
+    queryKey: ["web3hub/manifests", selectedCategory],
```

Comment:
I don't know if this is the only query you use, but `"web3hub/manifests"` could be part of an enum maybe as you might want to invalidate queries later?

(it's just code polishing 🫣)

Reply:
Yep usually I export the queryKey with the hook, I'll update